tests/api/users.test.js

```diff
@@ -54,27 +55,171 @@ describe("POST /api/users/login", () => {
+  });
+});
+
+describe("POST api/users/local/register", () => {
+  let mockUser;
+
+  beforeAll(async () => {
+    await mongoMemoryServer.connect();
+    mockUser = {
+      uid: "652df1ac5bfa32a524169098",
+      password: "testpassword",
+      displayName: "Test User",
+      photoURL: "http://example/com/test.jpg",
+    };
+  });
+
+  afterAll(async () => {
+    await User.deleteMany({});
+    await mongoMemoryServer.closeDatabase();
+  });
+
+  it("returns 201 on successful registration", async () => {
+    const response = await request(app).post("/api/users/local/register").send({
+      uid: mockUser.uid,
+      password: mockUser.password,
+      displayName: mockUser.displayName,
+      photoURL: mockUser.photoURL,
+    });
+
+    const userFound = await User.find({ uid: "652df1ac5bfa32a524169098" });
+
+    expect(response.status).toEqual(201);
+    expect(response.body.result).toEqual("Success");
+    expect(userFound[0]).toBeDefined();
+    expect(userFound[0].uid).toEqual(mockUser.uid);
+    expect(userFound[0].name).toEqual(mockUser.displayName);
+    expect(userFound[0].photoURL).toEqual(mockUser.photoURL);
+  });
+});
+
+describe("POST /api/users/local/login", () => {
+  beforeAll(async () => {
+    await mongoMemoryServer.connect();
+
+    mockUser = {
+      uid: "652df1ac5bfa32a524169098",
+      password: "testpassword",
+      displayName: "Test User",
+      photoURL: "http://example/com/test.jpg",
+    };
+
+    const password = await bcrypt.hash(mockUser.password, 10);
+
+    await User.create({
+      uid: mockUser.uid,
+      password: password,
+      name: mockUser.displayName,
+      photoURL: mockUser.photoURL,
+    });
+  });
+
+  afterAll(async () => {
+    await mongoMemoryServer.closeDatabase();
+  });
+
+  afterEach(async () => {
+    await User.deleteMany({});
+  });
+
+  it("should return 201 and user info on successful login", async () => {
+    const response = await request(app).post("/api/users/local/login").send({
+      uid: mockUser.uid,
+      password: mockUser.password,
+    });
+
+    const userFound = await User.find({ uid: mockUser.uid });
+
+    expect(response.status).toBe(201);
+    expect(response.body.user.uid).toEqual(mockUser.uid);
+    expect(response.body.user.name).toEqual(mockUser.displayName);
+    expect(userFound).toBeDefined();
+  });
+
+  it("should return 400 on unsuccessful login", async () => {
+    const password = await bcrypt.hash(mockUser.password, 10);
+
+    await User.create({
+      uid: mockUser.uid,
+      password: password,
+      name: mockUser.displayName,
+      photoURL: mockUser.photoURL,
+    });
+
+    const response = await request(app).post("/api/users/local/login").send({
+      password: "wrongpassword",
+      uid: mockUser.uid,
+    });
+
+    expect(response.status).toBe(400);
+    expect(response.body.message).toEqual(
+      "No user with that email or password",
+    );
+  });
+});
+
+describe("DELETE /api/users/delete", () => {
+  let mockUser;
```

Comment:
```suggestion
  const mockUser;
```
재할당을 하지 않는데 let을 사용하신 이유가 있을까요?🤔